core/services/directrequestocr/drlistener_test.go

```diff
@@ -113,8 +113,9 @@ func PrepareAndStartDRListener(t *testing.T) (*DRListenerUniverse, *log_mocks.Br
 	return uni, log, runBeganAwaiter
 }
 
+var RequestID drocr_service.RequestID = reqID(123)
```

Comment:
This can potentially break concurrent tests due to the shared same ID.

Reply:
Every test gets a separate plugin and set of ORMs so we should be good.